icu.lua

```diff
@@ -104,7 +104,7 @@ files {
   "icu4c/source/common/utrie2.cpp",
   "icu4c/source/common/uvector.cpp",
   "icu4c/source/common/wintz.cpp",
-  "icu4c/source/data/icudt69_dat.c",
+  "icu4c/source/data/icudt70_dat.c",
```

Comment:
Can a comment be copy pasted from the last time this was run so the workflow is documented?

Reply:
I was planning on adding some docs to rtc 3rd party docs for icu. Should I stick a comment in the icu.lua file itself as well?

Reply:
@mfpdias I think both are good actually. Here's a previous commit that did the same https://github.com/Esri/icu/pull/11/commits/68e8dc2243f376782b764446347c02ac5528bae8. This commit can be amended with the doc and the commit the previous commit can be dropped to lower the amount of commits as well to make it a little cleaner.

Reply:
I cleaned up the history and copied the commit message explaining data updates from the last release.